### DISCUSSION TITLE
Ability to Sort Tab inside Tabs

### DISCUSSION THREAD

**Discussion by ManojKiranA** [filamentphp/filament#1302] [Ideas]
It would be nice to have a method something like `sortOrder` or `viewOrder` 

```
Tabs::make('Heading')
    ->tabs([
        Tabs\Tab::make('Label 3')
            ->schema([
                // ...
            ])->sortOrder(3),
        Tabs\Tab::make('Label 2')
            ->schema([
                // ...
            ])->sortOrder(2),
        Tabs\Tab::make('Label 1')
            ->schema([
                // ...
            ])->sortOrder(1),
    ])
```

**Comment by danharrin** [comment]
Hey, what's the reason you cant put the tabs in order in the PHP?

**Comment by ManojKiranA** [comment]
In the view record page I am using the tab to organize the data. So based in roles the tab will be `hidden`/`shown `and also the order of the tab needs to be changed. because i have roughly 5 or 6 tabs. So by default filament allows me to conditionally hide the tab based on condition. So i thought it would be nice to have out of the box support for `sorting`/`reordering` the tabs
